### ISSUE TITLE
Fix adaptive card flickering for `updateActivity` in "team" scope

### ISSUE THREAD

**Issue by omikader** [MicrosoftDocs/msteams-docs#9391]
## Summary

I am using the JS Bot Framework SDK to build a Teams chatbot that responds to user queries in a "streamed" manner. In other words, I send an Adaptive Card using the `sendActivity` API and then make a streaming call to a 3rd party API which responds to my request in chunks. As chunks arrive, I call the `updateActivity` API using the initial activity's ID to update the card content.

As you can see below, this behavior works as expected in the "personal" scope (DMing the bot).

https://github.com/MicrosoftDocs/msteams-docs/assets/16735699/211ac539-9f63-4a70-838b-2724075f3b57

However, when I invoke the bot in a "teams" conversation, each `updateActivity` call causes the entire adaptive card to re-render which creates an undesirable UX due to the constant flickering/flashing.

https://github.com/MicrosoftDocs/msteams-docs/assets/16735699/52dff4ec-fbd3-4b2b-aaec-8a20bf98f5b6

### Code

```
import { ActivityType, AdaptiveCards, CardFactory } from "botbuilder";
import rawCard from "./adaptiveCards/answer.json" assert { type: "json" };

const attachment = CardFactory.AdaptiveCard(
    AdaptiveCards.declare<{answer: string}>(rawCard).render({ answer });
);

await context.updateActivity({
    type: ActivityTypes.MessageUpdate,
    id: "<original message ID>",
    attachments: [attachment],
});
```

I've also tried the [syntax](https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/update-and-delete-bot-messages?tabs=typescript#update-cards) from the documentation that uses the `MessageFactory` class which also does not work.

### TL;DR

**Is there a way I can solve for the flickering behavior when calling `updateActivity` for an Adaptive Card in a teams conversation?**

### Versions

`botbuilder`: 4.20.0
`@microsoft/adaptivecards-tools`: 1.3.2
Adaptive Card: 1.4
Microsoft Teams: 1.6.00.22155
NodeJS: v18.16.0

**Comment by Nivedipa-MSFT** [comment]
@omikader - Thanks for reporting your issue. We will test this and update you soon.

**Comment by omikader** [comment]
Hi @Nivedipa-MSFT, thanks for the response. Are there any updates on this issue? Thanks again!

**Comment by omikader** [comment]
@Nivedipa-MSFT are any updates on this? Thank you!
